include/SFML/Window/Joystick.hpp

```diff
@@ -41,12 +41,11 @@ namespace sf::Joystick
 /// \brief Constants related to joysticks capabilities
 ///
 ////////////////////////////////////////////////////////////
-enum
-{
-    Count       = 8,  //!< Maximum number of supported joysticks
-    ButtonCount = 32, //!< Maximum number of supported buttons
-    AxisCount   = 8   //!< Maximum number of supported axes
-};
+// NOLINTBEGIN(readability-identifier-naming)
+static constexpr unsigned int Count{8};
+static constexpr unsigned int ButtonCount{32};
+static constexpr unsigned int AxisCount{8};
```

Comment:
```suggestion
static constexpr int ButtonCount{32};
static constexpr int AxisCount{8};
```
I think these ought to be signed since they're compared against signed ints

Reply:
Yes, that's also why the CI fails.

Although the proper fix would be to change the use-sites, not the declarations --these should probably be `std::size_t` or at least `unsigned int` (counts).

Reply:
> Although the proper fix would be to change the use-sites

Agreed. I chose to recommend the simpler solution for the sake of moving this PR along faster.

Reply:
Also fine with me 🙂

Reply:
> these should probably be `std::size_t`

I agree, but the changes would propagate widely.